src/test/resources/testData/meth_with_cit.csv

```diff
@@ -0,0 +1,22 @@
+meth_cd,meth_nm
+BAROM,"Atmospheric pressure, barometer"
+ENCD ,"Gage height, shaft encoder"
+STRAN,"Gage height, subm press trans"
+WLNRG,"GW level, nonrecording gage"
+WLPRG,"GW level, pressure gage"
+WLACO,"GW level, acoustic pulse"
+WLCPG,"GW level, calib pressure gage"
+WLGPS,"GW level, differential GPS"
+WLSTP,"GW level, steel tape"
+WLAIR,"GW level, airline"
+WLCAI,"GW level, calib airline"
+WLUPT,"GW level,unvented press transdcr"
+WLLOG,"GW level, logs"
+HPENC,High precision encoder
+WLMAN,"GW level, manometer"
+SC001,Specific conductance sensor
+WLETP,"GW level, electric tape"
```

Comment:
@duselmann so on this line, for instance, the AQTS monitoring method is `GW level, electric tape` and that maps to method code `WLETP`.  This seems to be the case with the tables in the obsverations databases' tables that have `meth` somewhere in the name.  However, in the RDB generator, only 1 character is allowed for the method code in the RDB header file I believe (`lev_meth_cd` - `1ss`).  So knowing that, a few questions:

1. Is the RDB header correct (1ss vs. maybe 5ss)?
2. If the RDB header is correct, then this mapping I've made is definitely wrong.
3. If the RDB header is correct, then does the information to map these monitoring method strings in AQTS to NWIS single character codes exist somewhere?
4. Thanks!

Reply:
![image](https://user-images.githubusercontent.com/218559/95507374-f3711f00-0976-11eb-8292-ac47454022cc.png)